.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,6 +13,4 @@
 ...Any comments about the acceptance criteria that has been met or missed
 
 ### Issue
-...Include a link to the associated Issue Number, use "Resolves" to close the issue when the pull request is merged.
-
-`e.g. Resolves #123`
\ No newline at end of file
+Resolves #000
```

Comment:
```suggestion
Resolves #000
or
Related to #000
```

Reply:
resolves is 'dangerous' to have as the default. we could lose in progress tickets.